server/src/index.js

```diff
@@ -0,0 +1,6 @@
+const app = require('./app');
+
+app.listen(app.get('port'), () => {
+  // eslint-disable-next-line no-console
+  console.log('http://localhost:5000');
```

Comment:
this needs to be something like this.
```js
  console.log(`http://localhost:${app.get('port')}`);
```
suggession: define a const that takes app.get('port') and use it in the 2 places